nipoppy/data/examples/sample_pipelines/fmriprep-20.2.7/descriptor.json

```diff
@@ -588,9 +597,9 @@
     ],
     "tags": {},
     "suggested-resources": {
-        "cpu-cores": 8,
-        "ram": 8,
-        "walltime-estimate": 480
+        "cpu-cores": 1,
+        "ram": 1,
+        "walltime-estimate": 60
```

Comment:
These are the default values from Boutiques, agreed that they are too low but not sure what to put instead since depending on the invocation the timing will differ 🤷‍♀️